ingestion/tests/cli_e2e/base/config_builders/builders.py

```diff
@@ -53,14 +53,42 @@ def __init__(self, config: dict, config_args: dict) -> None:
         self.profilerSample = self.config_args.get("profilerSample", 100)
 
     # pylint: enable=invalid-name
-
     def build(self) -> dict:
         """build profiler config"""
         del self.config["source"]["sourceConfig"]["config"]
         self.config["source"]["sourceConfig"] = {
             "config": {
                 "type": "Profiler",
-                "generateSampleData": True,
+                "profileSample": self.profilerSample,
+            }
+        }
+
+        if self.config_args.get("includes"):
+            self.config["source"]["sourceConfig"]["config"]["schemaFilterPattern"] = {
+                "includes": self.config_args.get("includes")
+            }
+
+        self.config["processor"] = {"type": "orm-profiler", "config": {}}
+        return self.config
+
+
+class AutoClassificationConfigBuilder(BaseBuilder):
+    """Builder class for the AutoClassification config"""
+
+    # pylint: disable=invalid-name
+    def __init__(self, config: dict, config_args: dict) -> None:
+        super().__init__(config, config_args)
+        self.profilerSample = self.config_args.get("profilerSample", 100)
```

Comment:
Do we want 100% profile sample for tests?

Reply:
keeping what we had in the profiler already. I believe tables should be small. If we want to change this, let's change it everywhere and double-check the resulting counts